css/app/build.css

```diff
@@ -13,5 +13,9 @@
   background-color: #fe5857;
   color: #fff;
   border-radius: 22px;
-  box-shadow: 0px 2px 2px 0px rgba(0,0,0,0.2);
+  box-shadow: 0px 2px 2px 0px rgba(0, 0, 0, 0.2);
+}
+
+.notification-read .notification-badge {
```

Comment:
What's this meant to do? Note that if there's a notification inbox in the app, the notification inbox will also have an `app/build.css` file on the page, which may be before/after this file.